### DISCUSSION TITLE
how can we financially support the maintenance of cpal?

### DISCUSSION THREAD

**Discussion by louis030195** [tomaka/cpal#1328] [General]
From @roderickvd:

> Personally, I appreciate any [donations](https://github.com/sponsors/roderickvd) precisely as a token of appreciation. I don't depend on it, and it does not buy any promises or time away from my full-time job, but it does provide a lot of motivation and encouragement.

It also helps fund recurring software development licenses.

Every Euro is appreciated ❤️

**Comment by est31** [comment]
I am currently employed as an full time employee and don't intend to walk on the freelancer OSS maintainer path, simply because the expected revenue is not sufficient compared to what I would make in the industry.

**Comment by est31** [comment]
Thanks for the gesture though, and definitely appreciate it. if there is other folks with the right expertise who are interested in helping out for payment, I can help with coordination.

**Comment by roderickvd** [comment]
@louis030195 thank you for putting this on the table. In what capacity would you propose that? There's funding someone to do it for a living, then there's gestures of appreciation and encouragement, and anything in between.

Personally, I appreciate any [donations](https://github.com/sponsors/roderickvd) precisely as a token of appreciation. I don't depend on it, and it does not buy any promises or time away from my full-time job, but it does provide a lot of motivation and encouragement.

There's a lot of other OSS contributors with sponsorship pages that I'm sure will equally appreciate it.

Or were you thinking of "going big"? Interested to hear what you were thinking of.
